tests/apps/test_neural_net.py

```diff
@@ -14,40 +14,40 @@ class NeuralNetTester(tc.app.Library):
     @staticmethod
     def uses():
         return {
-            "ML": tc.hosted_ml.service.ML
+            "ML": tc.ml.service.ML
         }
 
     @tc.post
     def test_cnn_layer(self, cxt, inputs: tc.tensor.Tensor, labels: tc.tensor.Tensor) -> tc.F32:
-        layer = tc.hosted_ml.nn.ConvLayer.create([3, 5, 5], [2, 1, 1])
-        cxt.optimizer = tc.hosted_ml.optimizer.GradientDescent(layer)
+        layer = tc.ml.nn.ConvLayer.create([3, 5, 5], [2, 1, 1])
+        cxt.optimizer = tc.ml.optimizer.GradientDescent(layer)
         return cxt.optimizer.train(1, inputs, labels)
 
     @tc.post
     def test_cnn(self, cxt, inputs: tc.tensor.Tensor, labels: tc.tensor.Tensor) -> tc.F32:
         layers = [
-            tc.hosted_ml.nn.ConvLayer.create([3, 5, 5], [2, 1, 1], activation=tc.hosted_ml.sigmoid),
-            tc.hosted_ml.nn.ConvLayer.create([2, 3, 3], [2, 1, 1])
+            tc.ml.nn.ConvLayer.create([3, 5, 5], [2, 1, 1], activation=tc.ml.sigmoid),
+            tc.ml.nn.ConvLayer.create([2, 3, 3], [2, 1, 1])
```

Comment:
I think, We have to test it with Adam optimizer.

Reply:
This is a good point, I just found a bug where `Sequential.operator` loses track of the `Variable`s in layer 0 because of the `Tensor.concatenate` in layer 1

Reply:
I updated the test of a `Sequential` network of `ConvLayer`s to use `Adam` and added a custom `Operator` definition to support backpropagation (I'm not sure if the math is correct but we should be able to tell with further testing)